app/controllers/users_controller.rb

```diff
@@ -74,11 +79,10 @@ def parse_roster_ids(attrs)
   end
 
   def require_admin_in_roster_or_self
-    unless @current_user == @user || @current_user.admin_in?(@roster)
-      # ... and return is correct here
-      # rubocop:disable Style/AndOr
-      head :unauthorized and return
-      # rubocop:enable Style/AndOr
-    end
+    return if @current_user == @user || @current_user.admin_in?(@roster)
+    # ... and return is correct here
+    # rubocop:disable Style/AndOr
+    head :unauthorized and return
```

Comment:
Can you have two return statements in a row like this? Won't it just return and then never reach the `head :unauthorized` bit?